.github/workflows/ci.yml

```diff
@@ -14,7 +14,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        node-version: [12.x, 14.x, 15.x]
+        node-version: [14.x, 15.x, 16.x]
```

Comment:
Could you add 17.x too? See the maintenance release schedule - 17.x will be maintenance LTS from April so we should include it.

To future proof I would also try testing 18.x now and if it works include it, because that will be active LTS in October this year.

![image](https://user-images.githubusercontent.com/31989238/155710156-ec946346-284f-449a-9b2b-9e94570e9805.png)

Reply:
@MeStrak I added 17.x.

18.x is not yet supported by github actions.